resen-core/resources/helpers/setup_py27_env.sh

```diff
@@ -70,4 +73,4 @@ conda remove --quiet --yes -n py27 --force qt pyqt
 conda clean -tipy
 rm -rf ~/.cache/pip/*
 
-conda deactivate
```

Comment:
Why is this being removed?

Reply:
`conda deactivate` is not being removed. I think its the ending of the line is what is different, maybe because I edit it on my local machine and then push to my fork repository. I'll see if I can avoid that change.

Reply:
Oh, my bad. Nevermind then.

Reply:
In the original file, the last line did not ended in \n. The new file ends with an \n which I think is best practices.

Reply:
That's a PEP-8 thing, yes.